ScnSoft-Education/LeetCodeTraining.Tests/SortColorsTest.cs

```diff
@@ -0,0 +1,16 @@
+﻿using LeetCodeTraining.SortColors;
+
+namespace LeetCodeTraining.Tests
+{
+    public class SortColorsSolutionTest
+    {
+        [Theory]
+        [InlineData(new int[] { 2, 0, 2, 1, 1, 0 }, new int[] { 0, 0, 1, 1, 2, 2 })]
+        [InlineData(new int[] { 2, 0, 1 }, new int[] { 0, 1, 2 })]
+        public void SortColors(int[] input, int[] expected)
+        {
+            SortColorsSolution.SortColors(input);
```

Comment:
Please add // Act // Assert comments and extra line for clarity and similarity with others test and then please complete the PR.